e2e-tests/functions

```diff
@@ -366,7 +366,7 @@ run_pumba() {
 deploy_cert_manager() {
     kubectl_bin create namespace cert-manager || :
     kubectl_bin label namespace cert-manager certmanager.k8s.io/disable-validation=true || :
-    kubectl_bin apply -f https://github.com/jetstack/cert-manager/releases/download/v0.14.1/cert-manager.yaml --validate=false || : 2>/dev/null
+    kubectl_bin apply -f https://github.com/jetstack/cert-manager/releases/download/v0.15.1/cert-manager.yaml --validate=false || : 2>/dev/null
 }
 
 destroy() {
```

Comment:
You might need to upgrade the destroy function in the tests to version v0.15.1 too.

Reply:
Hi @boTux 

thank you for your feedback, I have fixed deletion of cert-manager and added check if `Issuer` and `Certificates` was created by cert-manager correctly